skops/tests/test_persist.py

```diff
@@ -0,0 +1,338 @@
+    "IsotonicRegression()",
+    "IterativeImputer()",
+    "KBinsDiscretizer()",
+    "KNeighborsClassifier()",
+    "KNeighborsRegressor()",
+    "KNeighborsTransformer()",
+    "KernelCenterer()",
+    "KernelDensity()",
+    "LabelBinarizer()",
+    "LabelEncoder()",
+    "Lars()",
+    "LarsCV()",
+    "LassoLars()",
+    "LassoLarsCV()",
+    "LassoLarsIC()",
+    "LatentDirichletAllocation()",
+    "LocallyLinearEmbedding()",
+    "MLPClassifier()",
+    "MLPRegressor()",
+    "MiniBatchDictionaryLearning()",
+    "MultiLabelBinarizer()",
+    "MultiOutputClassifier(estimator=LogisticRegression(C=1))",
+    "MultiOutputRegressor(estimator=Ridge())",
+    "NearestNeighbors()",
+    "NeighborhoodComponentsAnalysis()",
+    "OneHotEncoder()",
+    "OneVsOneClassifier(estimator=LogisticRegression(C=1))",
+    "OneVsRestClassifier(estimator=LogisticRegression(C=1))",
+    "OrdinalEncoder()",
+    "OrthogonalMatchingPursuit()",
+    "OrthogonalMatchingPursuitCV()",
+    "OutputCodeClassifier(estimator=LogisticRegression(C=1))",
+    "PLSCanonical()",
+    "PLSRegression()",
+    "PLSSVD()",
+    "PassiveAggressiveClassifier()",
+    "PatchExtractor()",
+    "Perceptron()",
+    "PoissonRegressor()",
+    "RFE(estimator=LogisticRegression(C=1))",
+    "RFECV(estimator=LogisticRegression(C=1))",
+    "RadiusNeighborsClassifier()",
+    "RadiusNeighborsRegressor()",
+    "RadiusNeighborsTransformer()",
+    "RandomForestClassifier()",
+    "RandomForestRegressor()",
+    "RandomTreesEmbedding()",
+    "RegressorChain(base_estimator=Ridge())",
+    "SGDClassifier()",
+    "SGDOneClassSVM()",
+    "SelectFdr()",
+    "SelectFpr()",
+    "SelectFromModel(estimator=SGDRegressor(random_state=0))",
+    "SelectFwe()",
+    "SelectKBest()",
+    "SelectPercentile()",
+    "SelfTrainingClassifier(base_estimator=LogisticRegression(C=1))",
+    "SequentialFeatureSelector(estimator=LogisticRegression(C=1))",
+    "SparseRandomProjection()",
+    "SpectralBiclustering()",
+    "SpectralEmbedding()",
+    "SplineTransformer()",
+    "StackingClassifier(estimators=[('est1',LogisticRegression(C=0.1)),('est2',LogisticRegression(C=1))])",  # noqa: E501
+    "StackingRegressor(estimators=[('est1',Ridge(alpha=0.1)),('est2',Ridge(alpha=1))])",
+    "TSNE()",
+    "TfidfTransformer()",
+    "TfidfVectorizer()",
+    "TweedieRegressor()",
+    "VotingClassifier(estimators=[('est1',LogisticRegression(C=0.1)),('est2',LogisticRegression(C=1))])",  # noqa: E501
+    "VotingRegressor(estimators=[('est1',Ridge(alpha=0.1)),('est2',Ridge(alpha=1))])",
+}
+
+
+def _tested_estimators(type_filter=None):
+    for name, Estimator in all_estimators(type_filter=type_filter):
+        try:
+            # suppress warnings here for skipped estimators.
+            with warnings.catch_warnings():
+                estimator = _construct_instance(Estimator)
+        except SkipTest:
+            continue
+
+        yield estimator
+
+    # nested Pipeline & FeatureUnion
+    # fmt: off
```

Comment:
Nice, hopefully this should work, I think we might need special data for these as well, but let's see.

Reply:
This case already works